### PR TITLE
styling. tools section styling + carousel fixes.

### PR DESCRIPTION
**Pull request by raydecastro**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -708,6 +708,9 @@ <h3 class="col-md-11 col-lg-11">Electrify</h3>
   <!-- end footer -->
   <script src="./scripts/mobile-pillars.js"></script>
   <script src="https://code.jquery.com/jquery-3.1.1.js" integrity="sha256-16cdPddA6VdVInumRGo6IbivbERE8p7CQR3HzTBuELA=" crossorigin="anonymous"></script>
+  <script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"
```
Comment (ananavati) [inline index.html]: why are we loading bootstrap at the bottom of the footer? there should be one load which I am assuming is happening already somewhere in the <head> tag?

Comment (raydecastro) [inline index.html]: @ananavati the only use we have to for the bootstrap.js that I'm aware of is the carousel. We put all the scripts at the bottom of the page, for optimized rendering of html and css content. We only load bootstrap.js once.

Comment (ananavati) [inline index.html]: yeah we should load external scripts from inside `<head>`

Comment (raydecastro) [inline index.html]: @ananavati jquery and bootstrap has been moved inside `<head>` section.

**Pull request merged**